lib/maintenance_mode_message.js

```diff
@@ -1,6 +1,5 @@
 var warning = [
-  'The AWS SDK for JavaScript (v2) will enter maintenance mode',
-  'on September 8, 2024 and reach end-of-support on September 8, 2025.\n',
+  'The AWS SDK for JavaScript (v2) is in maintenance mode.\n',
```

Comment:
"... in maintenance mode and will no longer release features for this service."

Reply:
The AWS SDK for JavaScript (v2) is in maintenance mode...
SDK releases are limited to address critical bug fixes and security issues only.

Reply:
```suggestion
  'The AWS SDK for JavaScript (v2) is in maintenance mode.',
  ' SDK releases are limited to address critical bug fixes and security issues only.\n',
```